android/src/main/java/com/gr4vy/embedreactnative/Gr4vyActivity.java

```diff
@@ -314,7 +317,8 @@ public void onStart() {
             requireSecurityCode,
             shippingDetailsId,
             merchantAccountId,
-            null, // TODO: pass connectionOptions
+            null,
```

Comment:
passing `null` for `connectionOptions` since we're using the string argument here